src/splitInteger.test.js

```diff
@@ -1,21 +1,55 @@
 'use strict';
 
 const splitInteger = require('./splitInteger');
+// const { parts } = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
+test(`should split a number into equal parts if a value
+ is divisible by  a numberOfParts`, () => {
+  expect(splitInteger(16, 4)).toEqual([4, 4, 4, 4]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const mas = [4];
 
+  expect(splitInteger(4, 1).length).toEqual(mas.length);
```

Comment:
In this case, it is better to hardcode the expected result as you did in the first test. Because here, you only check the length, but with the hardcoded value, you will additionally check the result's correctness.